index.html

```diff
@@ -58,9 +47,158 @@ <h1>Hey There. <br>I'm Mike</h1>
       </div>
 
     </div>
+  </section>
+
+  <section class="works">
+
+    
+    <div class="w1">
+
+      <h2 class="works-title">My Recent Works</h2>
+      <span class="line-works"></span>
+      
+      <img class="w1-img" src="img/w1.png" alt="w1" >
+      <p class="w1-title">Multi-Post Stories</p>
+      <p class="w1-text">
+      A daily selection of privately personalized reads; no accounts or sign-ups required.
+      has been the industry's standard dummy text ever since the 1500s, when an unknown
+      printer took a standard dummy text.</p>
+
+      <ul class="w1-lan">
+        <li>css</li>
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      
+      <p class="w1-see">See Project</p>
+
+    </div>
+
+    <div class="w2">
+      <div class="w2-project">
+        <p class="project-title">Profesional Art Printing Data</p>
+        <p class="project-text">
+          A daily selection of privately personalized reads; no accounts
+          or sign-ups required. has been the industry's standard
+        </p>
+
+        <ul class="project-lan">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+
+        <p class="project-see">See Project</p>
```

Comment:
- [ ] The button (e.g., "See project") should be a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))